sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/ReadmeSamples.java

```diff
@@ -94,8 +94,8 @@ public void createChatThread() {
     public void getChatThread() {
         ChatClient chatClient = createChatClient();
 
-        String chatThreadId = "Id";
-        ChatThread chatThread = chatClient.getChatThread(chatThreadId);
+        ChatThreadClient chatThreadClient = chatClient.getChatThreadClient("Id");
```

Comment:
Isn't ChatThreadClient created by the builder now? Please be consistent: eight getChatThreadClient from chatClient or get it from the builder NOT both

Reply:
The Azure review requested we also be able to create it with the builder, but no mention of exclusivity.

Reply:
The builder does not replace this method, all the other SDKs have this method, so I don't think we should remove it.
The Azure review board asked us to also be able to create it with the builder.

Reply:
Yes, this has been discussed in the arch board review - https://github.com/Azure/azure-sdk/issues/1895

Also, refer to the guideline here - https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-client-hierarchy